publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationFromImportCandidateHandlerTest.java

```diff
@@ -102,7 +101,7 @@ void shouldReturnPublicationResponseWhenPublicationHasBeenCreated() throws NotFo
         var updatedImportCandidate = importCandidateService.getImportCandidateByIdentifier(
                 importCandidate.getIdentifier());
 
-        assertThat(updatedImportCandidate.getImportStatus(), is(instanceOf(Imported.class)));
+        assertThat(updatedImportCandidate.getImportStatus().getCandidateStatus(), is(equalTo( CandidateStatus.IMPORTED)));
```

Comment:
I guess this was the only place when my suggestions made the code slightly simpler to read.